plsc/entry.py

```diff
@@ -408,7 +411,8 @@ def build_program(self,
                                                    param_attr=self.param_attr,
                                                    bias_attr=self.bias_attr,
                                                    margin=self.margin,
-                                                   scale=self.scale)
+                                                   scale=self.scale,
+                                                   data_format=self.data_format)
```

Comment:
Is it necessary to add the data_format parameter as the only value for it is self.data_format?